src/meta-srv/src/procedure/region_failover/update_metadata.rs

```diff
@@ -183,19 +200,29 @@ impl State for UpdateRegionMetadata {
         ctx: &RegionFailoverContext,
         failed_region: &RegionIdent,
     ) -> Result<Box<dyn State>> {
-        self.update_meta(ctx, failed_region).await.map_err(|e| {
-            RetryLaterSnafu {
-                reason: format!(
-                    "Failed to update metadata for failed region: {}, error: {}",
-                    failed_region, e
-                ),
-            }
-            .build()
-        })?;
+        self.update_metadata(ctx, failed_region)
+            .await
+            .map_err(|e| {
+                RetryLaterSnafu {
+                    reason: format!(
+                        "Failed to update metadata for failed region: {}, error: {}",
+                        failed_region, e
+                    ),
+                }
+                .build()
+            })?;
         Ok(Box::new(RegionFailoverEnd))
     }
 }
 
+fn table_metadata_lock_key(region: &RegionIdent) -> Key {
```

Comment:
Can we move all these distributed locking key generate functions into a single mod? It's much more readable and maintainable. The maintainer can read all the keys in one place.